conjur/constants.py

```diff
@@ -26,8 +26,8 @@
 DEFAULT_CERTIFICATE_FILE = os.path.expanduser(os.path.join('~', "conjur-server.pem"))
 CREDENTIAL_HOST_PATH = "/authn"
 
-PASSWORD_COMPLEXITY_CONSTRAINTS_MESSAGE = "The password must contain at least 12 characters: " \
-                                          "2 uppercase, 2 lowercase, 1 digit, 1 special character"
+PASSWORD_COMPLEXITY_CONSTRAINTS_MESSAGE = "(it must contain at least 12 characters: " \
```

Comment:
saving error messages as consts is very nice. do you think it will be more consistent if we move other messages from this PR as well? for example
`"Failed to execute command. Reason: The client was initialized without certificate verification, "
                            "even though the command was ran with certificate verification enabled. To continue "
                            "communicating with the server insecurely, run the command again with --insecure flag. "
                            "Otherwise, reinitialize the client.\n"`